java/src/jmri/jmrix/nce/NceConsistManager.java

```diff
@@ -59,13 +60,13 @@ public boolean csConsistNeedsSeperateAddress() {
      * Add a new NceConsist with the given address to consistTable/consistList
      */
     @Override
-    public Consist addConsist(DccLocoAddress locoAddress) {
+    public Consist addConsist(LocoAddress locoAddress) {
         if (consistTable.containsKey(locoAddress)) // no duplicates allowed.
         {
             return consistTable.get(locoAddress);
         }
         log.debug("Add consist, address " + locoAddress);
-        NceConsist consist = new NceConsist(locoAddress, memo);
+        NceConsist consist = new NceConsist((DccLocoAddress) locoAddress, memo);
```

Comment:
Before this cast, you need to make sure the address is a DCCLocoAddress.  You should throw an illegal argument exception if it isn't.